docs/plugins.md

```diff
@@ -203,4 +203,4 @@ Provided by [@jhildenbiddle](https://github.com/jhildenbiddle/docsify-tabs).
 
 ## More plugins
 
-See [awesome-docsify](awesome?id=plugins)
+See [awesome-docsify](https://github.com/docsifyjs/awesome-docsify/blob/master/README.md#plugins)
```

Comment:
Thank you for your contribution.

But this modification seems to be incorrect. 

Can't you access this link? https://docsify.js.org/#/awesome?id=plugins

Reply:
Hi @sy-records Yes I can access this link [docsify.js.org/#/awesome?id=plugins](https://docsify.js.org/#/awesome?id=plugins), should I update this link as the correct one in this change? Thanks for the correction.

Reply:
No, you don't have to revise.
If you can access it, it means the original link is working and I will close your PR.
Thanks again.